README.md

```diff
@@ -4,13 +4,19 @@ This repository contains data standards for Regen Network.
 
 ## Taxonomies
 
-Currently, there are two example taxonomies that can be managed in the following directories:
+Taxonomy cards are intended to provide a high-level sense of a project’s activities, environment, and impact. The labels and associated definitions and icons convey a general idea rather than provide an analytical understanding. For more details about a project’s activities, environment, and impact, it will be necessary to read the relevant project documentation.
+
+Taxonomies can be managed in the following directories dedicated to the three taxonomy card types:
 * Impacts: [/src/content/impact](./src/content/impact)
 * Activities: [/src/content/activity](./src/content/activity)
-* Ecosystems: [/src/content/ecosystem](./src/content/ecosystem)
+* Environment Type: [/src/content/environment-type](./src/content/environment-type)
+
+Each taxonomy card type can have hierarchical relationships, which will be useful when selecting, searching, and grouping terms on the web page. Below is an example. For example, if the top level of the hierarchy is “Marine,” and sub-categories under that are "Coastal", "Coral Reef", and "Deep Water". The terms for each of those levels needs a definition.
+
+In the above example, the Marine.md file would go into the “environment-type” directory (/src/content/environment-type/Marine.md) and the CoralReef.md file would go in the “Marine” directory (/src/content/environment-type/Marine/CoralReef.md)
 
```

Comment:
So in this model you could have the same sub directory items under different directory items, such as coastal for marine and coastal for wetlands. that doesn't present a problem though because it's organized under a different directory item, is that correct?

Reply:
The situation you describe is possible but my take is that it should be discouraged/rejected if someone proposed that. Unless, of course, we want coastal to show up under marine and wetlands. At this point, I'd say that's for the approvers to decide using "due diligence." I expect these sorts of questions will pop up from time to time.